tests/test_regressor/test_residuals.py

```diff
@@ -280,6 +280,33 @@ def test_residuals_plot(self):
 
         self.assert_images_similar(visualizer)
 
+    @pytest.mark.xfail(
+        IS_WINDOWS_OR_CONDA,
+        reason="font rendering different in OS and/or Python; see #892",
+    )
+    def test_residuals_plot_QQ_plot(self):
+        """
+        Image similarity of residuals and Q-Q plot on random data with OLS
+        """
+        _, ax = plt.subplots()
+
+        visualizer = ResidualsPlot(LinearRegression(), hist=False,
+                                                        qqplot=True, ax=ax)
+
+        visualizer.fit(self.data.X.train, self.data.y.train)
+        visualizer.score(self.data.X.test, self.data.y.test)
+        visualizer.finalize()
```

Comment:
@lwgray I recommend removing this call to finalize and seeing if that helps fix the tests on Travis - there is a known problem that the image similarity tests aren't removing text from the alternate axes (as you can see in the baseline image) - and text is one of the primary reasons we have a divergence between AppVeyor and Travis. 

Were the baseline images created on a Windows machine or Mac/Linux? That might also contribute to this; we have an xfail for windows or conda, but if the image were generated on Windows than it would fail anyway on Travis.

Reply:
@VladSkripniuk can you remove the call to finalize as described in the above comment from @bbengfort ?
I updated your branch so make sure you pull before you commit and new changes.
Also, did you create the baseline images on Windows or Mac?

Reply:
@lwgray Sure, I just removed the call to finalize. I created `tests/baseline_images/test_regressor/test_residuals/test_residuals_plot_QQ_plot.png` in commit 71f83bbf1ae314a5573e154aa7d92a24169816a3 using Ubuntu 18.04.

Reply:
Hi @VladSkripniuk your baseline image is out-of-date.  We have since merged fixes issues we were having with matplotlib,  so you will need to recreate your baseline images.  You can perform the following commands to reset your baseline images.

python -m tests.images -C tests/test_regressor/test_residuals.py
pytest -vs tests/test_regressor/test_residuals.py
python -m tests.images tests/test_regressor/test_residuals.py

Reply:
@lwgray I executed the commands you mentioned:
```
python -m tests.images -C tests/test_regressor/test_residuals.py
pytest -vs tests/test_regressor/test_residuals.py
python -m tests.images tests/test_regressor/test_residuals.py
```
 and ran `python -m pytest tests/test_regressor/test_residuals.py` and tests passed successfully on my machine. I upload the updated version of `tests/baseline_images/test_regressor/test_residuals/test_residuals_plot_QQ_plot.png`

Reply:
@VladSkripniuk great call on upgrading mpl to 3.2.1 - I forgot to tell you to do that.